core/objects_test.go

```diff
@@ -144,3 +144,23 @@ func TestAuthorizationSolvedBy(t *testing.T) {
 		})
 	}
 }
+
+func TestChallengeStringID(t *testing.T) {
+	ch := Challenge{
+		Token: "asd",
+		Type:  ChallengeTypeDNS01,
+	}
```

Comment:
I hate to be a stickler about something minor but could you add another testcase with the same token but a different challenge type so that there's something checking that both fields are being hashed?

Reply:
Yorp.